src/Core/Domain/Product/QueryResult/CategoriesInformation.php

```diff
@@ -31,34 +31,34 @@
 /**
  * Contains information about product categories
  */
-class ProductCategoriesInformation
+class CategoriesInformation
```

Comment:
BC Break : the class exists in 178x

Reply:
It's only used in experimental page so acceptable, but it's worth mentioning it in the PR description indeed

Reply:
@zuk3975 as you hesitated in your PR description You can indeed mark it as containing BC breaks, but they are acceptable so you just need to add a paragraph describing what was "broken"